examples/playlist/schemas/kind.json

```diff
@@ -0,0 +1,36 @@
+{
+  "group": "playlists.ext.grafana.com",
+  "kind": "Playlist",
+  "description": "A set of dashboards that will be displayed in a loop (dummy for testing)",
+  "maturity": "merged",
+  "machineName": {
+    "plural": "playlists",
+    "singular": "playlist"
+  },
+  "versions": [
```

Comment:
How strong / provable / reproducible is this definition? 🤔 

Does this pull request adds some kind of "framework", "similar to Thema" to ensure the correctness there? Is it delegated to "santosh" lib? Could you point to that or clarify it, for the sake of reviewing the pull request, please? Thanks!

Reply:
this PR is exploring how we can prove something works (or not) by having a set of tests and examples that we expect to be valid or invalid.

The actual ability to migrate up+down is hugely dependant on context, and I don't believe any system that claims to do this without human validation.

> delegated to "santosh" lib?
just the json validation part, not the migration part

Reply:
I agree with what regards to the implementation details, but I suspect will need to have a single source of truth for those schemas (registry?) and that's the reason behind my initial question.

I see this as plugin version, and its manifest details (Grafana version compat), or like Grafana semver, I strongly suspect we'll need a mechanism to keep a coherence and with some guarantees across all kind schemas. Otherwise, it'd be almost impossible to build something strong on top of it, no matter what underlying technology we use.